### DISCUSSION TITLE
modification needed in OpenSSL package for arm64 kernel build for android

### DISCUSSION THREAD

**Discussion by swinder0161** [Homebrew/discussions#853] [Everyday usage] [answered]
Hi Admins,

Please let me know how I can contribute to modifying the OpenSSL package install script and make it compatible to link headers and dylib at the right place

Currently, I do it manually on each OpenSSL upgrade

1)
cd /usr/local/include
ln -s ../Cellar/node/15.9.0/include/node/openssl ./

2)
cd /usr/local/lib
ln -s ../Cellar/openssl@1.1/1.1.1i/lib/libcrypto.dylib ./

**Comment by Bo98** [comment, marked as answer]
If you know what you are doing, you can use `brew link` to force symlink OpenSSL to `/usr/local/lib`. We do not do that automatically for OpenSSL since it can break applications which look for the `libcrypto.dylib` included with macOS (though this is less of an issue since 10.15).

Instead of `brew link` however, I suggest passing in LDFLAGS to whatever you are building. Just about all build systems allow you to do that. You can pass in the directory `/usr/local/opt/openssl/lib`, which won't break between version updates.

**Reply by swinder0161** [reply]
I think I can live with using /usr/local/opt/openssl/[lib,include]
